### ISSUE TITLE
Not enough VRAM available

### ISSUE THREAD

**Issue by TSUNAMIZREZ** [UnamSanctam/UnamWebPanel#135]
![image](https://user-images.githubusercontent.com/105847294/169713967-5ffaaf95-1b59-4b08-a9df-0967f9c980ed.png)
Hi everything works on my pc but people who launch it appear on the site but display "Not enough VRAM available" while people have good graphics cards and processors

**Comment by UnamSanctam** [comment]
Show what GPU they have, they need to have at least 4.9GB of free VRAM to be able to mine ETH.

**Comment by TSUNAMIZREZ** [comment]
![ZePhWX7](https://user-images.githubusercontent.com/105847294/170275624-7509f3d2-5c1d-4225-9649-ec916c561dda.gif)
So I have to turn to another cryptocurrency for mining on less powerful computers than you recommend for small office PCs ?

**Comment by UnamSanctam** [comment]
Yes, you can try ETC (Ethereum Classic) which only requires 3GB of VRAM compared to ETH's 4.9GB.

**Comment by UnamSanctam** [comment]
No, no (big) CPU coin is more profitable than a (big) GPU coin. Monero uses the RandomX algorithm which is a CPU "only" algorithm (it's possible to mine with the GPU but it's not profitable by design).
